packages/graphback-datasync/tests/DataSyncMongoDBTests.ts

```diff
@@ -0,0 +1,108 @@
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import { ObjectID, MongoClient } from 'mongodb';
+import { buildSchema } from 'graphql';
+import { filterModelTypes, GraphbackCoreMetadata } from '@graphback/core';
+import { advanceTo, advanceBy } from "jest-date-mock";
+import { SchemaCRUDPlugin } from "@graphback/codegen-schema";
+import { DataSyncMongoDBDataProvider, DataSyncPlugin, createDataSyncMongoCRUDRuntimeContext } from '../src';
+
+export interface Context {
+    providers: { [modelname: string]: DataSyncMongoDBDataProvider };
+    server: MongoMemoryServer
+  }
+  
+  export async function createTestingContext(schemaStr: string, config?: { seedData: { [collection: string]: any[] } }): Promise<Context> {
```

Comment:
I remember seeing this function somewhere, it'll probably be useful to extract this somewhere

Reply:
Not to be done in this PR of course.